### ISSUE TITLE
Windows installation insufficiencies

### ISSUE THREAD

**Issue by a1880** [MiniZinc/MiniZincIDE#89]
During my installation attempts for IDE 2.3.0, I noticed:

The program is installed in/under a default path without asking.
This used to be better and more flexible in previous releases. It would be helpful to allow more than one release in parallel. At least, the installer should allow for a custom installation path.

The Windows Registry settings on Minizinc IDE are partly inconsistent.
Old release settings are left behind side-by-side with the new settings.
It would be good to check the Registry during installation and fix or at least warn about outdated settings.

Thank you for your work!

Axel Kemper

**Comment by guidotack** [comment]
Apparently this behaviour changed on Windows 10. I have added a setting to our installation scripts to always show the directory selector.
For the registry cleanup, we'll have to postpone that to a later release (this seems to be a bit more complicated since it's all handled through the Qt configuration system). Could you let me know which particular settings are causing problems?

**Comment by a1880** [comment]
Sorry, but I removed the inconsistent registry trees without backup to start a fresh install. They probably were not actually causing problems. It just looked strange to have various releases in parallel, partly with different formats. In hindsight, the problem was that an executable of Google OR Tools appeared in two different places, and the subsequent error message of minizinc.exe made the IDE stop execution.

**Comment by a1880** [comment]
Installation path is now properly selectable. I am using Windows 10 with MiniZinc IDE 2.3.1
Registry tree looks consistent.

Issue is ready for closing. Thank you!
